django/db/backends/base/schema.py

```diff
@@ -97,6 +99,7 @@ def execute(self, sql, params=()):
                 "that can't perform a rollback is prohibited."
             )
         # Log the command we're running, then run it
+        sql = force_text(sql)
```

Comment:
I suppose I could use `str()` instead of `force_text()` but I think we'll have to keep string conversion around.